### PR TITLE
update ja/docs/concepts/overview/what-is-kubernetes.md

### LINKED ISSUE #26872: ja: Make docs/concepts/overview/what-is-kubernetes.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/concepts/overview/what-is-kubernetes.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/concepts/overview/what-is-kubernetes.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/overview/what-is-kubernetes.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/overview/what-is-kubernetes.md

diff between translated and v1.19
https://gist.github.com/123880ea2648fdc0cd282eec60ae1552

**Note**
Currently, we use **dev-1.19-ja.1** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by s-kawamura-w664**
Updated ja/docs/concepts/overview/what-is-kubernetes.md to follow v1.19 of the original (English) text.
Fixes #26872

### CODE REVIEW

**Comment by atoato88** [comment]
/lgtm

It's a good translation about `share of CPU` because of some members discussed at [here](https://github.com/kubernetes/website/pull/24659#issuecomment-713707339), I think.

**Comment by s-kawamura-w664** [comment]
/assign @nasa9084

Please review and if it is OK, please approve.

**Comment by kakts** [comment]
ご対応ありがとうございます
/lgtm

File: content/ja/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ weight: 10
 card:
   name: concepts
   weight: 10
+sitemap:
+  priority: 0.9
```
[the remark this answers is not in the document]
Comment (s-kawamura-w664) [reply content/ja/docs/concepts/overview/what-is-kubernetes.md:12]: Priority will affect sitemap.xml.
In English, this was modified in PR: https://github.com/kubernetes/website/pull/24149 for Issue: https://github.com/kubernetes/website/issues/24148.
The default priority is set to 0.75 in config.toml.
config.toml is out of localization. Therefore, Japanese pages will be affected.
I think other files should also be modified follow English.
What do you think?

Target files:
* ja/_index.html
* ja/docs/_index.md
* ja/docs/concepts/overview/_index.md

[the remark this answers is not in the document]
Comment (inductor) [reply content/ja/docs/concepts/overview/what-is-kubernetes.md:12]: discussed with others and it looks fine. thank you!

**Pull request merged**